jwst/resample/resample_step.py

```diff
@@ -88,6 +89,11 @@ def process(self, input):
                 raise RuntimeError(f"Input {example_model} is not a 2D image.")
             del example_model
 
+            # Make sure all input models have consistent NaN and DO_NOT_USE values
+            for model in input_models:
```

Comment:
@melanieclarke I think these are the lines you were talking about in the stand-up.  The syntax looks correct.  If you wanted to, you could add a delete of `model` at the end, outside the `for` loop like I show in the suggestion, but I doubt it would matter much.

I think this is in an ok location too.  Ideally we would minimize the number of times models are iterated over because each of these is potentially a load and save operation when `on_disk` is True, but in this case I don't see a good place for this code inside a pre-existing loop in `ResampleData`.

That was a long way of saying LGTM! :)

Reply:
Thank you for reviewing again, hopefully for the last time!